Stanislav_Hryhorskiy/HW6/hw6_practical_task_1.py

```diff
@@ -0,0 +1,22 @@
+"""
+*** Practical task 1 ***
+In the range from 1 to 10 determine:
+• even numbers that are divisible by 2;
+• odd numbers, which are divisible by 3;
+• numbers that are not divisible by 2 and 3.
+"""
+
```

Comment:
output your program:
In the range from 1 to 10:
        • even numbers that are divisible by 2: 2, 4, 6, 8, 10;
        • odd numbers, which are divisible by 3: 3, 6, 9;      
        • numbers that are not divisible by 2 and 3: 1, 5, 7.
6 is in first and second list. It is bad solution. Please try again do this task

Reply:
Thank you. I realized my mistake.